content/articles/deploying-a-serverless-django-restapi-with-zappa-on-aws/index.md

```diff
@@ -0,0 +1,449 @@
+
+Thereby, the user does not have to worry about manual configuration of servers.
+
+*Isn't this cool?*
+
+The main purpose for the development of serverless technology was to aid developers build and run applications without having to interact or manage servers
+
+Now, this does not mean the application runs entirely without a server, but that the management of the server would be handled by Amazon Web Services (AWS) through an open-source project known as [Serverless Application Model (SAM) framework](https://github.com/aws/serverless-application-model).
+
+One of the upsides of serverless technology is that it runs on pay-per *request*.
+
+For example, a new company or startup that is low on budget can host their web apps on [AWS Lambda](https://aws.amazon.com/lambda/) to pay only when a user sends a request to the web app.
+
+This process is quite exciting although the implementation in Django may be challenging. However, this is made easy using ***Zappa***.
+
+[Zappa](https://www.zappa.io/) is an open-source tool for developing, deploying, and maintaining serverless Python applications on Amazon Web Services (AWS) technologies including Lambda and [API Gateway](https://aws.amazon.com/api-gateway/).
+
+To aid your understanding, [AWS Lambda](https://aws.amazon.com/lambda/) is an Amazon serverless computing platform, referred to as the crux of AWS serverless offering that enables the user to run functions and manage computer resources automatically.
+
+These configurations and deployment are handled automatically by Zappa with just a click. Well, not literally you will still need to configure it by running Zappa in it and setting a few parameters using a CLI interface.
+
+However, this is a lot easier and faster than conventional web servers.
+
+In the course of this project, we will work on how Zappa is implemented in Django by building a serverless Django REST API and deploying the application to AWS Lambda using Zappa.
+
+### Prerequisites
+- Basic knowledge of Python.
+- Good understanding of Django.
+- [An AWS account](https://aws.amazon.com/).
+
+### How does Zappa work
+Zappa is a tool that allows deploying Django applications to AWS in a serverless environment. With Zappa, developers can deploy a [WSGI-compatible](en.wikipedia.org/wiki/Web_Server_Gateway_Interface) application using Amazon Web Service products such as Lambda, API Gateway, and S3.
+
+When the user sends a request to the server that is received by API Gateway, it starts a server inside our lambda function. Now, Django is installed in the same way that a lambda function is. The request is subsequently sent to the server, which the Django app handles via the WSGI layer. The server transmits the response to the API Gateway shortly before it is destroyed. Lastly, the API Gateway provides the client with a response.

```

Comment:
Could you brief more about this? 

If possible, you can write these as bulleted points.